sample_client_api/bootup/nvidia_objects.py

```diff
@@ -30,10 +30,12 @@ def initialize_nvidia_service() -> NvidiaImageGenerationTaskHandler:
 class BootupManager:
     def __init__(self):
         self.nvidia_task_handler: NvidiaImageGenerationTaskHandler = None
-        self.perform_bootup()
```

Comment:
If you don't perform bootup all the auth classes and nvidia classes won't load

Reply:
it's being performed twice. once in fastapi construction and once here on class init.

Reply:
it didn't matter before because there was no state but now it would leak the connection pool.